Dockerfile

```diff
@@ -1,21 +1,5 @@
-ARG GOVERSION=1.19
-
-FROM golang:${GOVERSION}-alpine as builder
-
-WORKDIR /build
-
-RUN apk add --no-cache make git bash
-
-ADD go.mod .
-ADD go.sum .
-ADD Makefile .
-RUN make dep
-
-ADD . .
-RUN make build
```

Comment:
I think with that we should either add a stage to Makefile to address the build locally first and then do `docker build` or keep the full docker build (the current version) in root and have the other (copy binary only) in `deployments/` or `/build` dir? 🤔

Reply:
Makes sense. Went with a different `Dockerfile` for release inside `deployments`.